index.js

```diff
@@ -263,6 +267,7 @@ function SequestPut (conn, opts, path) {
   }
 
   this.connection = conn
+  this.connection.on('error', this.emit.bind(this, 'error'))
```

Comment:
Move this to line `275` so we dont end up attaching more than one listener to this if we get an already established connection.